src/main/java/io/jenkins/plugins/zerobug/ZeroBugPublisher.java

```diff
@@ -133,7 +133,8 @@ public void perform(Run<?, ?> run, FilePath workspace, Launcher launcher, TaskLi
 		}
 	}
 
-	@Symbol("ZeroBugPublisher")
+	@Symbol({"zeroBug", "ZeroBugPublisher"}) // Prefer zeroBug, accept ZeroBugPublisher as keyword
+

```

Comment:
Remove the inserted empty line
```suggestion
	@Symbol({"zeroBug", "ZeroBugPublisher"}) // Prefer zeroBug, accept ZeroBugPublisher as keyword
```